src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2s.rs

```diff
@@ -47,25 +78,70 @@ pub fn t2s() -> CustomCpuTemplate {
                 subleaf: 0x0,
                 flags: KvmCpuidFlags(1),
                 modifiers: vec![
+                    // EBX:
+                    // - Bit 02: SGX
+                    // - Bit 04: HLE
+                    // - Bit 06: FDP_EXCPTN_ONLY
+                    // - Bit 09: Enhanced REP MOVSB/STOSB
+                    // - Bit 11: RTM
+                    // - Bit 12: RDT-M
+                    // - Bit 13: Deprecates FPU CS and FPU DS
+                    // - Bit 14: MPX
+                    // - Bit 15: RDT-A
+                    // - Bit 16: AVX512F
+                    // - Bit 17: AVX512DQ
+                    // - Bit 18: RDSEED
+                    // - Bit 19: ADX
+                    // - Bit 21: AVX512_IFMA
+                    // - Bit 22: PCOMMIT (deprecated)
```

Comment:
According to the last spec I could get from the [Intel SDM](https://www.intel.com/content/www/us/en/developer/articles/technical/intel-sdm.html), Bit 22 is just `Reserved`. Is `PCOMMIT` relevant to specific generations of CPUs? (I also did not check AMD's specs)